docs/public/pub-faq.rst

```diff
@@ -39,3 +39,5 @@ What about re-enabling support for other OSes and other platforms than Linux x86
 ------------------------------------------------------------------------------------
 
 We doubt that 32-bit platforms will be supported. Regarding other operating systems, we do not have resources to implement it on our own. However, patches are welcome.
+
+**UPD:** Since January 2020 build for macOS 10.13+ is supported.
```

Comment:
It's better to say here that build on both Linux and macOS 10.13+ is supported. The date when it was achieved is not important, plus if we eventually add more platforms, we won't need "UPD2"

Reply:
Removed completely. Let's rephrase the thing about multi-OS support in `INSTALL.rst`.